oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -942,6 +950,10 @@ Builder setEnvironmentProvider(EnvironmentProvider environmentProvider) {
       return this;
     }
 
+    public HttpTransportFactory getHttpTransportFactory() {
```

Comment:
Why was this added here?

Reply:
1. Following the same practice as `ServiceAccountCredentialsTest` check, it's exposed from `ServiceAccountCredentials`.
2. In the concern of any subclass which will also expose `transportFactory` in future and they are following the same logic.